app/Models/MemoryNote.php

```diff
@@ -28,8 +34,25 @@ class MemoryNote extends BaseModel
         'body',
     ];
 
+    // == logging
+
+    public function getActivitylogOptions(): LogOptions
+    {
+        return LogOptions::defaults()
+            ->logOnly([
+                'user_id',
+            ])
+            ->dontSubmitEmptyLogs();
```

Comment:
Something about this is causing log entries to be generated that only contain the user_id when modifying code notes created by the user.
```
{"attributes":{"user_id":43495},"old":{"user_id":43495}}
```

Reply:
Good catch, fixed in latest with the addition of a `->logOnlyDirty()`.